### ISSUE TITLE
Initialize counters rotation f

### ISSUE THREAD

**Issue by sleemans** [FirstLegoLeague/fllscoring#165]

**Comment by rikkertkoppes** [comment]
Is this related to #39 ? This seems to split up in two

- sliders are not correctly reset to their "undefined" value
- it is sometimes desirable to reset to "default" values, as indicated by #39. However, this may lead to errors.

Let this issue focus on the first problem

**Comment by poelstra** [comment]
Yes, related to #39, and this issue was according to @sleemans indeed mostly about the not resetting-part.

I've also heard people 'complain' today about not having the defaults being selected on an empty sheet.
When I explained about the potential for errors, most grumbled a bit at first, but then agreed it doesn't save that much time to have the defaults, and does make things 'safer'.
I think we could improve on this if we actively mark blocks as 'done' (e.g. light green background) when they're completed.

**Comment by rikkertkoppes** [comment]
That would be a nice one (marking blocks as done). Is a nice helper when hunting for missions that are not yet filled (when error is encountered at bottom)

**Comment by sleemans** [comment]
@rikkertkoppes it would be a clear marker to make finished missions slightly green.
